src/dbt_score/dbt_utils.py

```diff
@@ -31,8 +42,13 @@ def dbt_parse() -> dbtRunnerResult:
     Raises:
         DbtParseException: dbt parse failed.
     """
+    if not DBT_INSTALLED:
+        raise DbtNotInstalledException(
+            "To execute dbt parse, install dbt-core in your environment."
```

Comment:
Not sure if we plan to have more features requiring dbt, but if so, we can also consider making a decorator `@dbt_required` and implement this check to avoid repetitions. 
However, with 2 occurrences now seems to be to much